nasa-gibs/worldview
# WV-2374: React Map Component

## Description

This is an updated branch of [the original React map component PR](https://github.com/nasa-gibs/worldview/pull/4027) that includes the latest changes from our develop branch. I made a new branch so that I can preserve the old one in case we run into any issues. I tested this pretty thoroughly but it's probably worth one more look. 

## How To Test
1. `git checkout wv-2374-updated`
2. `npm ci`
3. `npm run build`
4. `npm run watch`
5. Everything should work the same. You should be able to see all of our changes we made in develop. There should be no performance issues.
